jbmc/regression/jbmc-strings/Makefile

```diff
@@ -28,4 +28,4 @@ show:
 clean:
 	find -name '*.out' -execdir $(RM) '{}' \;
 	find -name '*.gb' -execdir $(RM) '{}' \;
-	$(RM) tests.log
+	$(RM) tests.log tests-symex-driven-loading.log
```

Comment:
This actually isn't part of my PR, some random commits ended up in here because I rebased against the PR dependency and develop. Sorry.. it's ok if this doesn't get reviewed until the other is merged and I can rebase properly